docker/centos/build.Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM centos:latest
```

Comment:
why is this file called `build.Dockerfile`? Is it a common convention? (I'm not a docker expert)

Reply:
I'll rename it to Dockerfile.build, short google points that this is some kind of convention